drivers/stepper/step_dir/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 Fabian Blatz
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+zephyr_library_property(ALLOW_EMPTY TRUE)
```

Comment:
this is probably not needed since there's some files enabled unconditionally

Reply:
Done.